conjure-java-core/src/integrationInput/java/com/palantir/product/EmptyUnionTypeExample.java

```diff
@@ -68,18 +78,18 @@ public String toString() {
 
     private static final class VisitorBuilder<T>
             implements UnknownStageVisitorBuilder<T>, Completed_StageVisitorBuilder<T> {
-        private Function<String, T> unknownVisitor;
+        private BiFunction<String, Object, T> unknownVisitor;
 
         @Override
-        public Completed_StageVisitorBuilder<T> unknown(@Nonnull Function<String, T> unknownVisitor) {
+        public Completed_StageVisitorBuilder<T> unknown(@Nonnull BiFunction<String, Object, T> unknownVisitor) {
```

Comment:
I think this stage should continue to support `Function<String, T>` in addition to `BiFunction<String, Object, T>` -- that way folks who already use the staged builder aren't broken by setting the feature flag.

The implementation of `unknown(@Nonnull Function<String, T> unknownVisitor)` should delegate to `unknown(@Nonnull BiFunction<String, Object, T> unknownVisitor)`, ignoring the value input.

Reply:
that makes sense, let's break as few people as possible